addons/fries/XEH_postInit.sqf

```diff
@@ -4,7 +4,7 @@ if (is3DEN || !(EGVAR(common,aceFastroping))) exitWith {};
 
 if (isServer) then {
     ["Helicopter", "init", {
-        if (!GVAR(autoAdd)) exitWith {};
+        if ((!GVAR(autoAdd)) || (!(isNumber ((configOf (_this select 0)) >> QACEGVAR(fastroping,enabled))))) exitWith {};
         _this call ACEFUNC(fastroping,equipFRIES);
```

Comment:
Little bit too much parenthesis for my taste, I recommend to read up on operator precedence. Nice small change tho, should remove the warning spam in the 3den previews.
```suggestion
        params ["_vehicle"];
        if (!GVAR(autoAdd) || {!isNumber (configOf _vehicle >> QACEGVAR(fastroping,enabled))}) exitWith {};
        _vehicle call ACEFUNC(fastroping,equipFRIES);
```